src/HaskellWorks/Data/Dsv/Strict/Cursor/Type.hs

```diff
@@ -15,6 +18,9 @@ data DsvCursor t s = DsvCursor
   , dsvCursorNewlines  :: !s
   , dsvCursorPosition  :: !Word64
   }
+  deriving Generic
 
 deriving instance (Eq   (Elem t), Eq   t, Eq   s) => Eq   (DsvCursor t s)
 deriving instance (Show (Elem t), Show t, Show s) => Show (DsvCursor t s)
+
+instance (NFData t, NFData (Elem t), NFData s) => NFData (DsvCursor t s)
```

Comment:
How does this instance get used?

Reply:
`bench/main.hs` line 74, `nfIO` requires it since I changed the return type of `loadHwsvLazyIndex`